tests/common/database_trx_integration.cpp

```diff
@@ -22,6 +22,8 @@
 
 #include "database_trx_integration.hpp"
 
+#define SCORUM_NUM_INIT_DELEGATES 1
```

Comment:
Move to defines.hpp near:

#define TEST_INIT_DELEGATE_NAME "initdelegate"
#define TEST_INIT_KEY TEST_INIT_DELEGATE_NAME